app/Resources/views/pagelayout.html.twig

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en-US"
+      class="no-js">
+    <head>
+        {% if content is defined and title is not defined %}
+            {% set title = ez_content_name(content) %}
+        {% endif %}
+        <title>{{ title|default('Home')|trans }} - {{ 'eZ Platform' }}</title>
+
+        <meta http-equiv="Content-Type"
+              content="text/html; charset=utf-8" />
```

Comment:
Since this is an HTML5 document you don't have to use ending slashes in single html tags - here and below (ending slashes were required for XHTML documents).